R-package/DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: xgboost
 Type: Package
 Title: Extreme Gradient Boosting
-Version: 0.6.4.8
-Date: 2017-12-05
+Version: 0.7-0
```

Comment:
I thought we have agreed to an x.y.z version format https://xgboost.readthedocs.io/en/latest/how_to/contribute.html#r-package-versioning 
I admit, I've missed a chance to bump the R-package version to 0.7.0.1 when the core changed to 0.7, but I think it would be fair to start with 0.7.0 for the CRAN release.